static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-support-baseline.js

```diff
@@ -93,11 +93,26 @@ const changeHeaderLinks = (
     changeLinks('js-change-subscribe-link', subscribeLink);
 };
 
+const changeSideMenuLinks = (
+    becomeSupporterLink: string,
+    subscribeLink: string
+) => {
+    const cssClass = 'js-change-membership-item';
+
+    [...document.getElementsByClassName(cssClass)].forEach(el => {
+        if (el instanceof HTMLAnchorElement) {
+            if (el.innerText && el.innerText.trim() === 'become a supporter') {
+                el.href = becomeSupporterLink;
+            } else if (el.innerText && el.innerText.trim() === 'subscribe') {
+                el.href = subscribeLink;
+            }
+        }
+    });
+};
```

Comment:
Would it be possible to wrap the whole block into `fastdom.write(), since there is a lot of modification`? Also: the return type is missing.

```js
fastdom.read(() => document.getElementsByClassName(cssClass))
  .then(els => fastdom.write(() => {
    [...els].forEach(el => /*...*/)
  }));

Reply:
Updated in cff28c5185f01a05b7d6a1824f80982fcbdd0bf2